democracy/models/section.py

```diff
@@ -49,6 +49,8 @@ class Section(Commentable, StringIdBaseModel):
     content = models.TextField(verbose_name=_('content'), blank=True)
     plugin_identifier = models.CharField(verbose_name=_('plugin identifier'), blank=True, max_length=255)
     plugin_data = models.TextField(verbose_name=_('plugin data'), blank=True)
+    plugin_source = models.URLField(verbose_name=_('plugin source code'), blank=True)
```

Comment:
Actually, I should rename `plugin_identifier`to `plugin_type`since that is what it means.